test/system/505-networking-pasta.bats

```diff
@@ -0,0 +1,1348 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Networking with pasta(1)
+#
+# Copyright (c) 2022 Red Hat GmbH
+# Author: Stefano Brivio <sbrivio@redhat.com>
+
+load helpers
+
+
+### Addresses ##################################################################
+
+@test "podman networking with pasta(1) - IPv4 default address assignment" {
+	skip_if_no_pasta
+	skip_if_no_ipv4
+
+	run_podman run --net=pasta $IMAGE ip -j -4 addr show
+
+	output="$(ipv4_get_addr_global "${output}")"
```

Comment:
Please don't overload `$output`; use an actual real variable name instead. `$output` is a BATS necessity; in tests like these, actual useful variables are preferred.

Reply:
Hmm, this isn't just to recycle a variable. Logically, `$output` here should report the formatted IPv4 address. However, we don't have `jq` in the test image, so I'm deferring that part to the host. That is, I'm simply reformatting `$output` to its expected content. I tried coming up with a different name for it, but I'm failing... this is really meant to be `$output`!